packages/@aws-cdk/aws-batch/lib/job-queue.ts

```diff
@@ -0,0 +1,153 @@
+import { Construct, IResource, Resource, Stack } from '@aws-cdk/core';
+import { CfnJobQueue } from './batch.generated';
+import { ComputeEnvironment, IComputeEnvironment } from './compute-environment';
+
+/**
+ * Property to determine if the Batch Job
+ * Queue should accept incoming jobs to be queued
+ */
+export enum JobQueueState {
```

Comment:
Again - I don't see value in this enum, I'd rather it be an `enabled?: boolean` property.

Reply:
Got it. I will swap it out.